testit1a.chpl

```diff
@@ -0,0 +1,5 @@
+var tup = (1, 2.3, "hi");
```

Comment:
Should this be in the test/ directory somewhere?

Reply:
Oh shoot, thanks for catching that.  It should just be removed.  It was on my TODO list to remove, but I overlooked it when scanning the final PR.